tests/test_api_releases.py

```diff
@@ -0,0 +1,41 @@
+import pytest
+import json
+from fastapi.testclient import TestClient
+
+from main import app
+
+client = TestClient(app)
+
+expected_output_for_get_releases = {
+  "releases": [
+    {
+      "version": "2021.09",
+      "result": "In Progress",
+      "id": 3
+    },
+    {
+      "version": "2021.07",
+      "result": "Completed",
+      "id": 4
+    }
+  ]
+}
+
+def test_read_releases():
+    response = client.get("/releases")
+    assert response.status_code == 200
+    assert response.json() == expected_output_for_get_releases
+
+@pytest.mark.parametrize("release_id", [3, 4])
```

Comment:
Nit: I would rather this be 2 separate tests so we can see that the data we're asserting against in the test itself. It also gets rid of hardcoded index numbers.